samples/algorithms/repeat-until-success/RepeatUntilSuccess.qs

```diff
@@ -0,0 +1,110 @@
+namespace Microsoft.Quantum.Samples.RepeatUntilSuccess {
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Preparation;
+
+    /// Example of a Repeat-until-success circuit
+    /// The circuit is executed on a "target" qubit using an "ancilla" and 
```

Comment:
In general, Q# style has been to use "auxiliary" in strong preference to "ancillary" given the racial and misogynistic implications of the latter.

Reply:
Fixed!

Reply:
I did not know this, thanks for raising awareness Chris! I found this helpful explanation online as well: https://grammarist.com/usage/ancillary-vs-auxiliary/